### PR TITLE
Feat:added home page

### PR DESCRIPTION
**Pull request by talrejah02**
Added home page
Please review and give feedbacks
use this live [link](https://deploy-preview-1--curious-moonbeam-a53189.netlify.app/) to see the code

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
clean and modular code
UI looks good
suggestion: instead of adding images to repo you can host the images on Cloudinary and then use them 
looks good to merge

File: src/components/navbar/nav.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import Home from "../../assets/svg/Home.svg"
+import Explore from "../../assets/svg/Explore.svg"
+import Like from "../../assets/svg/Like.svg"
+import history from "../../assets/svg/History.svg"
+import playlist from "../../assets/svg/playlist.svg"
+import Logo from "../../assets/svg/Logo.svg"
+import account from "../../assets/svg/account.svg"
+import Setting from "../../assets/svg/Setting.svg"
```
Comment (gopalbbharadva) [inline src/components/navbar/nav.jsx:10]: Suggestion: You can create barrel export for SVGs.

**Review by gopalbbharadva — APPROVED** [review]
UI looks nice, Great job. good to go.

File: src/components/navbar/nav.jsx (hunk 1 above)
Comment (talrejah02) [reply src/components/navbar/nav.jsx:10]: thanks will work on your suggestion

**Pull request merged**